packages/core/src/local-admin-api.ts

```diff
@@ -20,13 +20,30 @@ export class LocalAdminApi implements AdminApi {
     return this.nodeStatusFn()
   }
 
-  async startIndexingModels(modelsIDs: Array<StreamID>): Promise<void> {
-    await this.indexApi.indexModels(modelsIDs)
-    await this.syncApi.startModelSync(modelsIDs.map((id) => id.toString()))
+  async startIndexingModels(
+    modelsIDs: Array<StreamID>,
+    indices?: Array<ModelFieldsIndex>
```

Comment:
why do we take both arguments here at all?  What do we do if they conflict?  Seems like we should just take a single arg that's an array of ModelFieldsIndex and it should be up to the calling code in the http-api to de-dupe and merge the arguments received on the endpoint.

Reply:
So this was to maintain backwards compatibility with the RemoteAdminApi, which ComposeDB uses. Both LocalAdminApi and RemoteAdminApi extend from AdminApi, where this method is defined.